### DISCUSSION TITLE
Does Minecraft need to be running? Does it still work in 1.16+? How long is the average wait time?

### DISCUSSION THREAD

**Discussion by NonNewtonianVoid** [Zodsmar/SeedSearcherStandaloneTool#86] [Q&A]
I just got the code and got it running. It ran for about 2 hr with no result. Is there something I'm doing wrong?
I had very specific settings and it was set for 1 seed...

**Comment by Zodsmar** [comment]
What were your settings? Without that info I have no idea. Some seeds are just physically impossible.
